openBCIBoard.js

```diff
@@ -83,37 +88,37 @@ function OpenBCIFactory() {
      * Author: AJ Keller (@pushtheworldllc)
      */
     OpenBCIBoard.prototype.connect = function(portName) {
-        var self = this;
-
         this.connected = false;
 
-        return new Promise(function(resolve,reject) {
+        return new Promise((resolve,reject) => {
```

Comment:
=> is a new ES6 feature. It binds this to the lexical this. Austin and I are are switching the whole library and will use babel to downgrade it to ES5 for support.